chandra_aca/tests/test_all.py

```diff
@@ -94,6 +94,21 @@ def test_mag_for_p_acq():
     assert np.allclose(mag, 10.821, rtol=0, atol=0.01)
 
 
+def test_halfwidth_adjustment():
+    for mag, mults in ((6.0, [1.0, 1.0, 1.0, 1.0]),
+                       (8.5, [1.0, 1.0, 1.0, 1.0]),
+                       (9.25, [0.9, 0.8, 0.7, 0.505]),
+                       (10.0, [0.8, 0.6, 0.4, 0.01]),
+                       (11.0, [0.01, 0.01, 0.01, 0.01])):
+        pacq = acq_success_prob(mag=mag, halfwidth=120)
```

Comment:
I'm pretty sure the boxes defined in characteristics are the max box sizes.  This reminds me, did you *really* want this to be stepwise? or would you want something closer to the 160 probability than the 120 probability to apply to a 155 box?  I thought that was a conscious decision on your part, but now I'm confused.